lib/chef/knife/bootstrap.rb

```diff
@@ -375,9 +375,7 @@ def run
             exit 1
           end
 
-          client_builder.run
-
-          chef_vault_handler.run(node_name: config[:chef_node_name])
+          chef_vault_handler.run(client_builder.run[:client])
```

Comment:
well I think this is somewhat different since this is a builder that we are treating like a client factory, but thats cool. I'll make the change.